src/components/Banner/index.tsx

```diff
@@ -0,0 +1,3 @@
+import styles from './index.module.scss'
+
+export default () => <div className={styles.Root} />
```

Comment:
Delete the styles related to `Mirana is alive`?

Reply:
Updated in https://github.com/nervosnetwork/ckb-explorer-frontend/pull/986/commits/86acefe9f58abf4aa894b222bf60ccb5c37c4ca5